neon_homeassistant_skill/vocab/en-us/lights.set.color.intent

```diff
@@ -1,2 +1,2 @@
-(set|change) (|the|my) {entity} color to {color}
-(set|change) (|the|my) light {entity} to {color}
\ No newline at end of file
+(set|make) (|the|my) {entity} color (to| ){color}
```

Comment:
I think you need a space after 'to'/outside the `()`

Reply:
I could swear I pit that after to, not after |...